android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -277,6 +295,8 @@ private void createViews() {
             CookieHandler.setDefault(DEFAULT_COOKIE_MANAGER);
         }
 
+        MultiDex.install(getContext());
```

Comment:
It looks strange to me.

Reply:
Hum… The fact that we need MultiDex?

Reply:
Maybe we don't need it, I just followed Google's documentation, I'll make all your suggestions then remove it to see if it still works

Reply:
Yes I understand but it should be called once in an application. Here you will call once per Video view (maybe multiple times on remount). I think this is more a responsibility of application.
Additionally you should include multidex dependency in gradle file if it is really mandatory.
But i think a note in documentation should be enough!

Reply:
Ok nice, I'll add this to the documentation and remove multidex from grade file and from here! 😉